djvm/src/test/kotlin/net/corda/djvm/execution/SandboxExecutorTest.kt

```diff
@@ -28,9 +28,8 @@ class SandboxExecutorTest : TestBase(KOTLIN) {
 
     @Test
     fun `can load and execute runnable`() = customSandbox(MINIMAL) {
-        val executor = DeterministicSandboxExecutor<Int, String>(configuration)
-        val summary = executor.run<TestSandboxedRunnable>(1)
-        val result = summary.result
+        val taskFactory = classLoader.createTypedTaskFactory()
+        val result =taskFactory.create(TestSandboxedRunnable::class.java).apply(1)
```

Comment:
A missing space between `=taskTaskFactory`. Who knows what the static code analysis might have to say about this.

Reply:
Thanks. The DJVM currently flies beneath the "DeteKt-ion Grid", but better safe than sorry!